addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -1250,6 +1256,36 @@ test("receive new chat messages: out of odoo focus (tab title)", async () => {
     await assertSteps(["set_title_part"]);
 });
 
+test("new message in tab title has precedence over action name", async () => {
+    const pyEnv = await startServer();
+    const bobUserId = pyEnv["res.users"].create({ name: "bob" });
+    const bobPartnerId = pyEnv["res.partner"].create({ name: "bob", user_ids: [bobUserId] });
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_type: "chat",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ partner_id: bobPartnerId }),
+        ],
+    });
+    mockService("presence", { isOdooFocused: () => false });
+    mockService("mail.out_of_focus", { setInterval: (func) => func() }); // so that no setInterval
+    await start();
+    await openDiscuss();
+    await contains(".o_breadcrumb:contains(Inbox)"); // wait for action name being Inbox
+    const titleService = getService("title");
+    expect(titleService.current).toBe("Inbox");
+    // simulate receiving a new message in chat 1 with odoo out-of-focused
+    await withUser(bobUserId, () =>
+        rpc("/mail/message/post", {
+            post_data: { body: "Hello world!", message_type: "comment" },
+            thread_id: channelId,
+            thread_model: "discuss.channel",
+        })
+    );
+    await contains(".o_notification:contains(Hello World!)");
+    expect(titleService.current).toBe("1 Message - Inbox");
```

Comment:
This assertion is failing because the mock of `setInterval` doesn't behave as if there were no `setInterval`. Quite the opposite, it actually speeds up the interval, making this test non-deterministic. If you were to disable it, it would look more like this:

```js
mockService("mail.out_of_focus", { setInterval: () => {} }); // so that no setInterval runs